src/code42cli/date_helper.py

```diff
@@ -18,7 +18,7 @@
 def convert_datetime_to_timestamp(dt):
     if dt is None:
         return
-    utc_epoch = datetime.utcfromtimestamp(0).astimezone(timezone.utc)
+    utc_epoch = datetime.utcfromtimestamp(0).replace(tzinfo=timezone.utc)
```

Comment:
I am surprised the tzinfo when doing utcfromtimestamp is not already set to UTC

Reply:
Yeah, it returns a naive dt object just with what the UTC value would be from epoch 0. The value doesn't ultimately change, but forcing the tzinfo on the object makes sure we don't make a wrong assumption later when interacting with it.